### ISSUE TITLE
Direction of line source as Green's function

### ISSUE THREAD

**Issue by hagenw** [sfstoolbox/sfs-matlab#122]
In `driving_function_imp_wfs_ls()` the line source has an orientation in the 3D case, which is provided by the user.
In `greens_function_imp()` this is not the case for the line source, see [line 97](https://github.com/sfstoolbox/sfs-matlab/blob/master/SFS_time_domain/greens_function_imp.m#L97). Should we include this there, too?

**Comment by trettberg** [comment]
Same thing with `driving_function_mono_wfs_ls()` and `greens_function_mono()`.

It would be nice to do this in a consistent way, but there will be some difficulties:

With the driving functions, `xs(3:6)` indicates orientation of the virtual line source.
For secondary sources on the other hand, `x0(3:6)` is needed for direction.

For the moment I think we should leave it as it is.

**Comment by hagenw** [comment]
That's true, but at least both driving functions and both Green's functions are consistent at the moment.

It's also ok for me to stay with this for the moment. If someone really needs this functionality she/he could change it then.
